pkg/controllers/secretstore/common.go

```diff
@@ -93,8 +97,8 @@ func validateStore(ctx context.Context, namespace string, store esapi.GenericSto
 	}
 	defer cl.Close(ctx)
 
-	err = cl.Validate()
-	if err != nil {
+	validationResult, err := cl.Validate()
+	if err != nil && !(validationResult == esapi.ValidationResultUnknown) {
```

Comment:
Just a tiny thing to make it slightly more readable
```suggestion
	if err != nil && validationResult != esapi.ValidationResultUnknown {
```

Reply:
indeed & fixed